waku_scalability/waku_scaling.md

```diff
@@ -232,10 +289,11 @@ Assumptions/Simplifications:
 - A03. The network topology is a d-regular graph of degree (static): 6
 - A41. Delay is calculated based on an upper bound of the expected distance.
 - A42. Average delay per hop (static): 0.1s.
+- A43. Average peer bandwidth (static): 30Mbps.
 
-For 100 the average latency is 0.257 s
-For 10k the average latency is 0.514 s (max with sharding)
-For  1m the average latency is 0.771 s (even in a single shard)
+For 100 the maximum latency is 0.301 s
+For 10k the maximum latency is 0.733 s (max with sharding)
+For  1m the maximum latency is 18.578  (even in a single shard)
```

Comment:
To understand, why we don't present also the average? I don't remember why we are changing average for maximum here.